packages/model-viewer/src/features/scene-graph/texture-info.ts

```diff
@@ -82,13 +82,13 @@ export class TextureInfo extends ThreeDOMElement implements
   onUpdate: () => void;
 
   constructor(
-      onUpdate: () => void, gltf: GLTF, material: MeshStandardMaterial|null,
-      texture: ThreeTexture|null, textureUsage: TextureUsage,
-      gltfTextureInfo: GLTFTextureInfo) {
+      onUpdate: () => void, gltf: GLTF,
+      material: Set<MeshStandardMaterial>|null, texture: ThreeTexture|null,
+      textureUsage: TextureUsage, gltfTextureInfo: GLTFTextureInfo) {
     super(
         onUpdate,
         gltfTextureInfo ? gltfTextureInfo : {index: -1},
-        new Set<ThreeTexture>([texture!]));
+        new Set<ThreeTexture>(texture ? [texture] : []));
```

Comment:
`$threeTexture` is duplicating `$source`; let's just use one; probably `$source` for consistency.

Reply:
done.